packages/tendermint-light-client/membership/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-//! The crate that contains the types and utilities for `sp1-ics07-tendermint-membership` program.
+//! The crate that contains the types and utilities for `tendermint-light-client-membership` program.
 #![deny(missing_docs, clippy::nursery, clippy::pedantic, warnings)]
```

Comment:
Can we add this as well
```suggestion
#![deny(missing_docs, clippy::nursery, clippy::pedantic, warnings, unused_crate_dependencies)]
```